src/org/spdx/html/LicenseJSONFile.java

```diff
@@ -79,7 +79,7 @@ protected JSONObject getJsonObject() {
 			}
 			jsonObject.put(SpdxRdfConstants.RDFS_PROP_SEE_ALSO, seeAlsoArray);
 		}
-		if (seeAlsos != null && seeAlsos.length > 0) {
+		if (crossRefs != null && crossRefs.length > 0) {
```

Comment:
This is already fixed in the code base - this may be showing as a diff due to the force push.